README.md

```diff
@@ -65,14 +64,6 @@ This is a project portraying the name of players and their respective scores in
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍

Reply:
Yeah sounds good to me too @HAMMAS-SALEEM